xfce/mousepad/Portfile

```diff
@@ -17,20 +17,20 @@ homepage            https://www.xfce.org
 master_sites        http://archive.xfce.org/src/apps/${name}/${branch}/
```

Comment:
Same comment as the other PR, let's have this URL be HTTPS (and any others that aren't currently).

Reply:
Done.